pkg/drivers/kic/types.go

```diff
@@ -26,15 +26,29 @@ const (
 	// Version is the current version of kic
 	Version = "v0.0.15-snapshot4"
 	// SHA of the kic base image
-	baseImageSHA = "ef1f485b5a1cfa4c989bc05e153f0a8525968ec999e242efff871cbb31649c16"
+	baseImageSHA          = "ef1f485b5a1cfa4c989bc05e153f0a8525968ec999e242efff871cbb31649c16"
```

Comment:
lets verify if SHA is same for arm64 and amd if not, we should have two shas in the code

Reply:
```
ubuntu@ip-172-31-39-30:~/src/minikube$ uname -a 
Linux ip-172-31-39-30 5.4.0-1032-aws #33-Ubuntu SMP Wed Dec 9 17:17:06 UTC 2020 aarch64 aarch64 aarch64 GNU/Linux
ubuntu@ip-172-31-39-30:~/src/minikube$ docker inspect --format='{{index .RepoDigests 0}}' ilyazuyev/kicbase:pr-test
ilyazuyev/kicbase@sha256:949338df4b157b907fbc9b27466c160c609f2f0f17c5dce73d2ceda084fedb69
```

```
izuyev@izuyev-macbookpro --- ~ » uname -a
Darwin izuyev-macbookpro.roam.corp.google.com 19.6.0 Darwin Kernel Version 19.6.0: Thu Oct 29 22:56:45 PDT 2020; root:xnu-6153.141.2.2~1/RELEASE_X86_64 x86_64
izuyev@izuyev-macbookpro --- ~ » docker inspect --format='{{index .RepoDigests 0}}' ilyazuyev/kicbase:pr-test
ilyazuyev/kicbase@sha256:949338df4b157b907fbc9b27466c160c609f2f0f17c5dce73d2ceda084fedb69
izuyev@izuyev-macbookpro --- ~ » 
```

It's the same sha1